### PR TITLE
begin migration to new eventual-send API

### PR DESCRIPTION
**Pull request by michaelfig**
Also, fix an infinite loop when pipelining property-gets.

### CODE REVIEW

File: src/kernel/liveSlots.js [hunk 1]
```diff
@@ -23,28 +23,19 @@ function build(syscall, _state, makeRoot, forVatID) {
...
     /* eslint-disable no-use-before-define */
     const handler = {
-      GET(target, prop) {
-        // Support: o~.[prop] remote property lookup
-        // FIXME: Do not stall the pipeline!
-        return target.then(o => o[prop]);
-      },
       POST(_o, prop, args) {
         // Support: o~.[prop](...args) remote method invocation
+        console.log(`queuing`, slot, prop);
```
Comment (warner) [inline src/kernel/liveSlots.js]: can we remove this console.log yet?

File: demo/encouragementBotCommsWavyDot/vat-bot.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function setup(syscall, state, helpers) {
       harden({
         encourageMe(nameObj) {
           // Test of tildot property get.
-          return nameObj~.name.then(name => {
+          return nameObj~.name~.identity().then(name => {
```
Comment (warner) [inline demo/encouragementBotCommsWavyDot/vat-bot.js]: The old code was exercising our ability to do `~.propname` GETs on a local object (the `{name: 'user'}` passed into `encourageMe()` would trigger the pass-by-copy behavior of marshal.serialize). This new code is not exercising that, but only GET on remote Presences.. should we find a way to retain that check?

File: src/kernel/liveSlots.js (hunk 1 above)
Comment (michaelfig) [reply src/kernel/liveSlots.js]: Definitely!

File: demo/encouragementBotCommsWavyDot/vat-bot.js (hunk 2 above)
Comment (michaelfig) [reply demo/encouragementBotCommsWavyDot/vat-bot.js]: I'll revert this, as it isn't even testing the case I wanted to.

**Pull request merged**